cannot find module ...config.json

after using with webpack/babel to build I'm getting the following error when running an electron app with this lib:

`App threw an error when running [Error: Cannot find module '/Users/bgrayburn/Library/Application Support/NewSapienceModelEditor/config.json'.]`

my understanding is this is coming from the require(file) line in the index.js of electron-json-config because such a config file module isn't loaded by webpack. I don't want to load the config file with webpack, but I do want to be able to use this lib with webpack.

My suggestion would be to use fs.readSync instead of require in the lib.  How does this sound?

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "electron-json-config",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "description": "Simply set and get configuration from a json file for your Electron app",
   "main": "src/index.js",
   "keywords": [

--- a/src/index.js
+++ b/src/index.js
@@ -8,7 +8,7 @@ const file = (electron.app || electron.remote.app).getPath('userData')+'/config.
 if(!exists.sync(file)) {
   fs.writeFileSync(file, '{}');
 }
-var config = require(file);
+var config = JSON.parse(fs.readFileSync(file));
 
 const sync = function() {
   fs.writeFileSync(file, JSON.stringify(config));

